sap_hana/datadog_checks/sap_hana/data/conf.yaml.example

```diff
@@ -0,0 +1,118 @@
+init_config:
+    ## @param global_custom_queries - list - optional
+    ## See `custom_queries` defined below.
+    ##
+    ## Global custom queries can be applied to all instances using the
+    ## `use_global_custom_queries` setting at the instance level.
+    #
+    # global_custom_queries:
+    #   - metric_prefix: ibm_db2
+    #     query: <QUERY>
+    #     columns: <COLUMNS>
+    #     tags: <TAGS>
+
+instances:
+
+    ## @param server - string - required
+    ## The hostname used to connect to the system.
+    #
+  - server: <SERVER>
+
+    ## @param port - integer - required - default: 39015
+    ## The port used to connect to the system.
+    #
+    port: 39015
+
+    ## @param username - string - required
+    ## The database user to authenticate as.
+    #
+    username: <USERNAME>
+
+    ## @param password - string - required
+    ## The password of `username`.
+    #
+    password: <PASSWORD>
+
+    ## @param use_hana_hostnames - boolean - optional - default: false
+    ## Whether or not to use the hostnames contained in HANA's monitoring
+    ## views instead of the Agent hostname.
+    #
+    # use_hana_hostnames: false
```

Comment:
why not both? We can have a hana_hostname tag

Reply:
Good idea 👍